xilem/src/lib.rs

```diff
@@ -139,6 +141,13 @@ pub trait WidgetView<State, Action = ()>:
     View<State, Action, ViewCtx, Element = Pod<Self::Widget>> + Send + Sync
 {
     type Widget: Widget;
+
+    fn flex(self, params: impl Into<FlexParams>) -> FlexItem<Self, State, Action>
```

Comment:
Having a builder method for this on the `WidgetViewTrait, is obviously open for discussion, I'm not sure of it yet:

What may or may not make sense from an API standpoint is to support these kinds of methods, when the returned element is also a `WidgetView`, `FlexItem` is not a `WidgetView`, so it can't be used outside of the context of a `Flex` container view.